nilearn/reporting/_utils.py

```diff
@@ -72,6 +73,9 @@ def clustering_params_to_dataframe(
         Its actual meaning depends on the height_control parameter.
         This function translates alpha to a z-scale threshold.
 
+    is_volume_glm: bool
+        True if we are dealing with volume data.
```

Comment:
Independent of your changes, the function has no default value for `min_distance`, but in the doc above, `min_distance : float, default=8` .